src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -594,24 +593,17 @@ SF10A::cycle()
 		return;
 	}
 
-
-	/* Trigger measurement */
-	if (OK != measure()) {
-		DEVICE_DEBUG("measure error lidar");
-	}
-
```

Comment:
Interesting, we might need to worry about the edge case when measure fails, we would want to try to set the bit again.